modules/testing/pages/integration/snapshots/index.adoc

```diff
@@ -93,17 +93,19 @@ It's possible to encounter a situation where new Snapshots cannot be created bec
 
 **Mitigation**: The primary way to prevent this is to manage the lifecycle of your Release CRs.
 
-**Configure the Grace Period**: Ensure the releaseGracePeriodDays field in your ReleasePlan is set to a reasonable value that reflects your team's release cadence. A shorter grace period ensures Release CRs expire more quickly, freeing up their associated Snapshots for garbage collection.
+**Configure the Grace Period**: Ensure the `gracePeriodDays` field for your Releases is set to a reasonable value that reflects your team's release cadence. This can be configured in your ReleasePlan and will be applied to created Release CRs. A shorter grace period ensures Release CRs are deleted more quickly, freeing up their associated Snapshots for garbage collection. KubeArchive uses a default of 5 days if not specified and enforces a maximum of 30 days at deletion time.
 
 **Manual Deletion**: In an urgent situation, you can manually delete old or unnecessary Release CRs to immediately make their Snapshots eligible for GC. 
 ====
 
 === Relationship with Releases
 
 * When you create a `Release` CR from a Snapshot, that Snapshot is protected from the count-based garbage collection described above *if the Release exists*.
-* `Release` CRs have their own time-based expiration, defined by the `releaseGracePeriodDays` field in the associated xref:releasing:create-release-plan.adoc[`ReleasePlan`] (default: **7 days**). This *is* configurable per `ReleasePlan`.
-* When a `Release` CR expires, it is automatically deleted.
-* Once the `Release` is deleted, its associated `Snapshot` CR is no longer protected and becomes eligible for count-based garbage collection if the retention limits are exceeded.
+* `Release` CRs have their own time-based expiration and automatic cleanup:
+** The retention period is defined by the `gracePeriodDays` field in the Release spec. This can be configured in the xref:releasing:create-release-plan.adoc[`ReleasePlan`] and will be applied to created Releases.
+** KubeArchive automatically deletes Release CRs from the cluster after their grace period expires based on the Release creation timestamp. If not specified, defaults to 5 days. KubeArchive enforces a maximum of 30 days at deletion time; higher values are treated as 30 days. No webhook validation prevents setting higher values in the ReleasePlan.
```

Comment:
Can we change `If not specified, defaults to 5 days` to `If no gracePeriodDays value is provided, KubeArchive will delete if after 5 days`?